frontend/src/pages/Settings/Settings.jsx

```diff
@@ -0,0 +1,69 @@
+import { Grid, Typography } from "@mui/material";
+import Avatar from "@mui/material/Avatar";
+import palette from "../../theme/palette.js";
+import typography from "../../theme/typography.js";
+import "./Settings.scss";
+import SettingsItem from "../../components/SettingsComponents/SettingsItem.jsx";
+import { ICON_STYLES } from "../../app/constants.js";
+import SettingsFooter from "../../components/SettingsComponents/SettingsFooter/SettingsFooter.jsx";
+import SettingsHeader from "../../components/SettingsComponents/SettingsHeader.jsx";
+
+const Settings = ({ isAuthUser = true }) => {
+  // дані для відображення(тест)
+  const languages = ["Українська", "English"];
+  const themes = ["Світла", "Темна"];
+
+  return (
+    <Grid
+      container
+      direction="column"
+      justifyContent="space-between"
+      height="100vh"
+      padding="8px 0"
+    >
+      <div>
+        <SettingsHeader title="Налаштування" dots={false} />
+
+        <Grid
+          container
+          justifyContent="start"
+          alignItems="center"
+          sx={{ marginTop: "13px", marginBottom: "13px", paddingLeft: "15px" }}
+        >
+          <Grid item>
+            <Avatar alt="userName" sx={{ width: 66, height: 66 }} />
+          </Grid>
+          <Grid item sx={{ marginLeft: "15px" }}>
+            <div className="settings-profile-name">
+              <Typography variant="h6" sx={{ flex: 1, ...typography.body1 }}>
+                Alina
+              </Typography>
+              <Typography variant="body2" sx={{ color: palette.grey[250] }}>
+                @jacob_d
+              </Typography>
+            </div>
+          </Grid>
+        </Grid>
+
+        <Grid>
+          <div className="separator" />
+          <SettingsItem
+            title="Мова інтерфейсу"
+            icon={ICON_STYLES.LANGUAGE}
+            props={languages}
+          />
+          <div className="separator" style={{ marginLeft: "64px" }} />
+          <SettingsItem title="Тема" icon={ICON_STYLES.THEME} props={themes} />
+          <div style={{ marginTop: "22px" }}>
+            <SettingsItem title="Правила спільноти" icon={ICON_STYLES.INFO} />
+          </div>
+        </Grid>
```

Comment:
Це я б виніс в окрему компоненту..
Бо тут трохи нагромаджено і багато пропсів((

  Або знову ж, щоб не виносити окремо.. то в SettingsItem можна зробити щоб передавати тільки 1 параметр, а все решту опрацьовувати "під капотом", в середині компоненти SettingsItem
```suggestion
        <Grid>
          <div className="separator" />
          <SettingsItem style={ICON_STYLES.LANGUAGE} />
          <div className="separator" style={{ marginLeft: "64px" }} />
          <SettingsItem style={ICON_STYLES.THEME}/>
          <div style={{ marginTop: "22px" }}>
            <SettingsItemstyle={ICON_STYLES.INFO} />
          </div>
        </Grid>
```
Приблизно так.. але тоді константу бажано перейменувати, або ж створити нову..
Продовжу цю ідею в інших коментаряхх